Makefile

```diff
@@ -4,66 +4,108 @@ VERSION=6.7
 DIST=$(NAME)-$(VERSION)
 
 SRC=sources
-FEA=$(SRC)/features
+FEADIR=$(SRC)/features
+GSUB=$(FEADIR)/gsub.fea
 DOC=documentation
 TOOLS=tools
 
 PY?=python
+PREPROP?=preprocess
 BUILD=$(TOOLS)/build.py
+LOADFEAT=$(TOOLS)/load-features.py
 NORMALIZE=$(TOOLS)/sfdnormalize.py
 CHECKERRS=$(TOOLS)/check-errors.py
 LO?=lowriter
 
 NULL=
 
-FONTS=Math-Regular \
-      Sans-Regular \
-      Sans-Bold \
-      Sans-Italic \
-      Serif-Regular \
-      Serif-Semibold \
-      Serif-Bold \
-      Serif-Italic \
-      Serif-SemiboldItalic \
-      Serif-BoldItalic \
-      SerifDisplay-Regular \
-      SerifInitials-Regular \
-      Mono-Regular \
-      Keyboard-Regular \
-      $(NULL)
+MFONTS=Sans-Regular \
+       Sans-Bold \
+       Sans-Italic \
+       Serif-Regular \
+       Serif-Semibold \
+       Serif-Bold \
+       Serif-Italic \
+       Serif-SemiboldItalic \
+       Serif-BoldItalic \
+       SerifDisplay-Regular \
+       $(NULL)
+
+OFONTS=Math-Regular \
+       SerifInitials-Regular \
+       Mono-Regular \
+       Keyboard-Regular \
+       $(NULL)
+
+FONTS=$(MFONTS) \
+      $(OFONTS)
 
 SFD=$(FONTS:%=$(SRC)/$(NAME)%.sfd)
-NRM=$(FONTS:%=$(SRC)/$(NAME)%.nrm)
-CHK=$(FONTS:%=$(SRC)/$(NAME)%.chk)
+MNRM=$(MFONTS:%=$(SRC)/$(NAME)%.nrm)
+ONRM=$(OFONTS:%=$(SRC)/$(NAME)%.nrm)
+NRM=$(MNRM) $(ONRM)
+FEA=$(MFONTS:%=$(SRC)/$(NAME)%.fea)
+MCHK=$(MFONTS:%=$(SRC)/$(NAME)%.chk)
+OCHK=$(OFONTS:%=$(SRC)/$(NAME)%.chk)
+CHK=$(MCHK) $(OCHK)
 DUP=$(FONTS:%=$(SRC)/$(NAME)%.dup)
 LNT=$(FONTS:%=$(NAME)%.lnt)
-OTF=$(FONTS:%=$(NAME)%.otf)
+MOTF=$(MFONTS:%=$(NAME)%.otf)
+OOTF=$(OFONTS:%=$(NAME)%.otf)
+OTF=$(MOTF) $(OOTF)
 PDF=$(FONTS:%=$(DOC)/$(NAME)%-Table.pdf)
 OPDF=$(DOC)/Opentype-Features.pdf $(DOC)/Sample.pdf
 
+FEADEFS=
+
 export SOURCE_DATE_EPOCH ?= 0
 
 all: otf
 
 otf: $(OTF)
 doc: $(PDF) $(OPDF)
+feature-files: $(FEA)
```

Comment:
I don’t think we need this target.

Reply:
It's so that if you're editing the feature files and want to verify your changes you don't have to wait through generating otfs just to see the files as a side-effect.

Reply:
Still not sure what kind of changes one would like to see, but it doesn’t hurt either.